<?xml version="1.0"?>
<launch>

    <arg name="in_odom_topic" default="/odom"/>
    <arg name="in_depthmap_topic" default="/depthmap"/>
    <arg name="out_cloud_topic" default="/cloud_synced"/> 
   
    <node pkg="graph_slam" name="publish_pointcloud_from_depthmap_node" type="publish_pointcloud_from_depthmap_node" output="screen">
        <rosparam file="$(find graph_slam)/config/intrinsics.yaml"/>
        <rosparam file="$(find graph_slam)/config/pointcloud_filters.yaml"/>
        <param name="in_odom_topic" value="$(arg in_odom_topic)"/>
        <param name="in_depthmap_topic" value="$(arg in_depthmap_topic)"/>
        <param name="out_cloud_topic" value="$(arg out_cloud_topic)"/>
    </node>

</launch>
